conductor/server/config/config.properties

```diff
@@ -0,0 +1,36 @@
+# Database persistence model.  Possible values are memory, redis, and dynomite.
+# If ommitted, the persistence used is memory
+#
+# memory : The data is stored in memory and lost when the server dies.  Useful for testing or demo
+# redis : non-Dynomite based redis instance
+# dynomite : Dynomite cluster.  Use this for HA configuration.
+
+db=dynomite
+
+# Dynomite Cluster details.
+# format is host:port:rack separated by semicolon
+workflow.dynomite.cluster.hosts=mender-dynomite:8102:us-east-1c
```

Comment:
yeah, this is just to make the parser happy...you have to pass a region (a 'rack' in coductor-speak).

I could set it to `dummyregion` since `us-east` may be confusing